tpp-app/src/settings/SettingsNavigator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Settings from "./Settings"
+import Notifications from './Notifications';
+const Stack = createNativeStackNavigator();
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+export default function SettingsNavigator() {
+    return(
+        <Stack.Navigator intialRouteName="Settings" screenOptions={{ headerShown: true,
+            headerStyle: {height: 200},
+            headerTitleStyle: {
+              fontWeight: "800",
+              fontSize: 20,
+              fontFamily: "Avenir",
+            },
+          }}>
+             <Stack.Screen name={"Settings"} component={Settings} />
```

Comment:
I'd do the map `STACK_SCREEN` thing from `CalendarNavigator` here, in case something outside needs to access it